libbeat/dashboards/import_dashboards.go

```diff
@@ -448,16 +448,15 @@ func (imp Importer) ImportDir(dirType string, dir string) error {
 	// check if the directory exists
 	if _, err := os.Stat(dir); err != nil {
 		// nothing to import
-		fmt.Println("No directory", dir)
-		return nil
+		return fmt.Errorf("No directory %s", dir)
```

Comment:
We decided not to commit the index-pattern and generate it every time is needed. To generate the index-pattern for a Beat, we just need to call `make update`. So, in order to fix the test, we just need to find a way to call this before running test. @ruflin knows more here.

As not all the Beats have a dashboard in the latest [beats-dashboards 5.1.1](https://artifacts.elastic.co/downloads/beats/beats-dashboards/beats-dashboards-5.1.1.zip) (see missing `filebeat/dashboard`), but all of them must have an `index-pattern` directory, we can raise an error only if the `index-pattern` directory is not available, and ignore or use `INFO` level in case the`dashboard` directory is not available.

Thank you @autarch for raising all these issues. I agree that there are a few points that we need to reconsider. We are planning to migrate the `import_dashboards` script to the Beat, so the user doesn't have to run an extra step to import the dashboards. This would be a good opportunity to improve the way we import the dashboards. 

Let's wait for @ruflin to find a way to fix the test, and then we see how we can proceed with the PR.